src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -2030,32 +2040,41 @@ JNI_METHOD(void, write)
     }
 }
 
+CHIP_ERROR PutPreencodedInvokeRequest(app::CommandSender & commandSender, uint32_t endpointId, uint32_t clusterId,
```

Comment:
as above: pass in `CommandPathParams` and `ByteSpan` to this method to make it clearer.

Reply:
Done, thanks